org.gtk.Gtk3theme.Flat-Remix-GTK-Yellow-Dark-Solid.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "org.gtk.Gtk3theme.Flat-Remix-GTK-Yellow-Dark-Solid",
+  "branch": "3.22",
+  "runtime": "org.freedesktop.Platform",
+  "build-extension": true,
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
+  "appstream-compose": false,
+  "separate-locales": false,
+  "modules": [
+    {
+      "name": "Flat-Remix-GTK-Yellow-Dark-Solid",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -dm755 /usr/share/runtime/share/themes/Flat-Remix-GTK-Yellow-Dark-Solid/gtk-3.0",
+        "cp -a Flat-Remix-GTK-Yellow-Dark-Solid/index.theme /usr/share/runtime/share/themes/Flat-Remix-GTK-Yellow-Dark-Solid/",
+        "cp -a Flat-Remix-GTK-Yellow-Dark-Solid/gtk-3.0/** /usr/share/runtime/share/themes/Flat-Remix-GTK-Yellow-Dark-Solid/gtk-3.0"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/sp1ritCS/flat-remix-gtk.git"
```

Comment:
Should be a tagged release or a specific commit. The same thing for the other 39 PR's